### DISCUSSION TITLE
CTL+ALT+P and CTL+ALT+O are not working.

### DISCUSSION THREAD

**Discussion by hemu2k6** [codefori/vscode-ibmi#1357] [General]
CTL+ALT+P and CTL+ALT+O are not working. I am getting the error - The editor could not be opened due to an unexpected error: CPFA0A9. 
I am able to open the same member from object browser. Thinking the issue may be due to the extensions (.lib/.file/.mbr). Please let me know if any setting has to be changed to make it work. Thank you.

**Comment by chrjorgensen** [comment]
@hemu2k6 When specifying members in the dialog, you must enter the member in the format specified in the placeholder:

![billede](https://github.com/halcyon-tech/vscode-ibmi/assets/13275072/30849aa5-5312-4863-90b3-7970539c94f4)

I.e. LIBRARY/FILE/MEMBER.EXT - no `/` before the library name and no `.LIB`, `.FILE` or `.MBR`.

**Reply by hemu2k6** [reply]
@chrjorgensen, I tried this and the error still exists. Although I have mentioned the extension only for the member (.mbr) the extension is automatically getting added for the library (.lib) and source file (.file) while opening the member. Not sure if any setting needs to be changed.

**Reply by chrjorgensen** [reply]
@hemu2k6 Please provide a screenshot of your dialog and supply more debug information (IBM i Help view, `Report an issue` and copy-paste the issue content here).

**Reply by hemu2k6** [reply]
@chrjorgensen, This is not working for me due to multiple Auxiliary Storage Pools. When I prefixed the path with IASP, it's started working. Thank you.
